StudyHub/StudyHub.BLL/Services/Auth/PasswordService.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Microsoft.AspNetCore.Identity;
+using StudyHub.BLL.Services.Interfaces.Auth;
+using StudyHub.Common.DTO.AuthDTO;
+using StudyHub.Common.Exceptions;
+using StudyHub.Entities;
+
+namespace StudyHub.BLL.Services.Auth;
+public class PasswordService : IPasswordService
+{
+    private readonly UserManager<User> _userManager;
+    public PasswordService(UserManager<User> userManager)
+    {
+        _userManager = userManager;
+    }
+    public async Task<string> ForgotPassword(ForgotPasswordDTO dto)
+    {
+        var user = await _userManager.FindByEmailAsync(dto.Email);
+
+        if (user == null)
+            throw new NotFoundException($"User with this email does not exist. Email: {dto.Email}");
```

Comment:
better write message like $,,User with this email {your email} does not exist,,

Reply:
In all our project, we used to write error messages exactly like that, so I think it's better to leave it like that

Reply:
I think @fedyafedko's variant quite clarify, that's why don't need changes here.